### PR TITLE
Do not use access token when downloading PDF report and Minio is disabled (AST-44042)

### PR DESCRIPTION
**Pull request by checkmarx-kobi-hagmi**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

> When Min.io is not used, no need to use access token when downloading PDF report.

### References

> https://checkmarx.atlassian.net/browse/AST-44042

### Testing

> Test exists already.

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).
- [ ] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used

### CODE REVIEW

File: internal/wrappers/results-pdf-http.go [hunk 1]
```diff
@@ -108,10 +109,11 @@ func (r *PdfHTTPWrapper) CheckPdfReportStatus(reportID string) (*PdfPollingRespo
 	}
 }
 
-func (r *PdfHTTPWrapper) DownloadPdfReport(reportID, targetFile string) error {
+func (r *PdfHTTPWrapper) DownloadPdfReport(url, targetFile string) error {
 	clientTimeout := uint(downloadTimeout)
-	customURL := fmt.Sprintf("%s/%s/download", r.path, reportID)
-	resp, err := SendHTTPRequest(http.MethodGet, customURL, http.NoBody, true, clientTimeout)
+	useAccessToken := FeatureFlags[MinioEnabled]
+	resp, err := SendHTTPRequestByFullURL(http.MethodGet, url, http.NoBody, useAccessToken, clientTimeout, "", true)
```
Comment (OrShamirCM) [inline internal/wrappers/results-pdf-http.go:115]: The url includes the path?

Comment (checkmarx-kobi-hagmi) [reply internal/wrappers/results-pdf-http.go:115]: it's the url to the file on S3:
https://s3.dualstack.me-central-1.amazonaws.com/reports.uae.me-central-1-..........

File: internal/wrappers/results-pdf-http.go [hunk 2]
```diff
@@ -108,10 +109,11 @@ func (r *PdfHTTPWrapper) CheckPdfReportStatus(reportID string) (*PdfPollingRespo
 	}
 }
 
-func (r *PdfHTTPWrapper) DownloadPdfReport(reportID, targetFile string) error {
+func (r *PdfHTTPWrapper) DownloadPdfReport(url, targetFile string) error {
 	clientTimeout := uint(downloadTimeout)
-	customURL := fmt.Sprintf("%s/%s/download", r.path, reportID)
-	resp, err := SendHTTPRequest(http.MethodGet, customURL, http.NoBody, true, clientTimeout)
+	useAccessToken := FeatureFlags[MinioEnabled]
```
Comment (tamarleviCm) [inline internal/wrappers/results-pdf-http.go:114]: leave a TODO for Elchanan here to change to the FF call

**Review by tamarleviCm — COMMENTED** [review]
are you checked the both flows?

File: internal/wrappers/results-pdf-http.go (hunk 1 above)
Comment (OrShamirCM) [reply internal/wrappers/results-pdf-http.go:115]: But it can be differnet one if MINIO false, no?

Comment (OrShamirCM) [reply internal/wrappers/results-pdf-http.go:115]: casue it shoudn't be s3 in usual but checkmarx

File: internal/wrappers/results-pdf-http.go (hunk 2 above)
Comment (OrShamirCM) [reply internal/wrappers/results-pdf-http.go:114]: He didn't change it .. he didn't have conflicts

Comment (OrShamirCM) [reply internal/wrappers/results-pdf-http.go:114]: in Elchanan's pr

File: internal/wrappers/results-pdf-http.go (hunk 1 above)
Comment (OrShamirCM) [reply internal/wrappers/results-pdf-http.go:115]: I'm approved for not being a blocker but please don't merge without verifying it

**Review by OrShamirCM — APPROVED** [review]

**Review by OrShamirCM — APPROVED** [review]

**Pull request merged**